src/container/home/ConferenceCarousel.tsx

```diff
@@ -0,0 +1,105 @@
+import { Box, Card, BoxProps, Typography, Grid, Button, Stack } from '@mui/material';
```

Comment:
It is not a courasel and i am also not sure if it is going to be a Carousel, so maybe name it to something more genering like `LandingConfereces` and `LandingCities` . Landing refering to landing page.